client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java

```diff
@@ -180,6 +281,7 @@ public CentralDogmaBeanFactory(CentralDogma dogma, ObjectMapper objectMapper) {
                 buildQuery(settings),
                 jsonNode -> {
                     try {
+                        changeListener.accept(bean);
```

Comment:
Could we surround this sentence with another `try-catch` block so that a user does not get a misleading exception message `Failed to convert...`?

Reply:
nice catch, how about just move it outside to let it throw whatever exception received from accept?

Reply:
That sounds fine.